examples/aws-noaa-oisst-avhrr-only-example.ipynb

```diff
@@ -0,0 +1,936 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false"># Select Sea Surface temperature for a single data and region
# Region defined by setting mim, max coordinates with slice
Line #1.    subset = ds["sst"].sel(time="2019-05-15",lat=slice(-10,30),lon=slice(70,110))

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/asdi-examples/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>